### PR TITLE
Feat: Journal Creation API

### PR DESCRIPTION
**Pull request by ydymegan**
# Notion Task

**Journal Creation API:** https://www.notion.so/70204328dfbc4abfa76a39d9c3b50169?v=21f1019a9ecd49b791569d2ce69e5cc7&p=9508e9b67ce64a0e98198ed403350c6b
**Submit Button:** https://www.notion.so/70204328dfbc4abfa76a39d9c3b50169?v=21f1019a9ecd49b791569d2ce69e5cc7&p=a4606ae9af814ac09fef2abeccea667c

# Changes Made In This PR

- created useJournalEntry hook
- added the createJournalEntry API function
- included the submit button and header text using i18n on the journal entry page
- updated `en.json` with raw texts for journal-related pages
- created navigation back to landing page upon submission of entry

# Screenshots

**Before:**
image

**After:**
image
image

### CODE REVIEW

File: src/api/journal/createJournalEntry.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import uuid from '../../utils/uuid';
+import initialJournalState from '../../data/initialJournalState';
+import { JournalEntry } from '../../types/Journal';
+import { User } from '../../types/User';
+
+const createJournalEntry = async (args: {
+  user: User;
+  title: string;
+  content: string;
```
Comment (millasml) [inline src/api/journal/createJournalEntry.ts:11]: these should be optional too

File: src/data/initialJournalState.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "title": "",
+    "content": ""
+}
+
```
Comment (millasml) [inline src/data/initialJournalState.json:5]: i dont think this is needed, as you reassign them in `createJournalEntry` anyways

File: src/types/Journal.ts [hunk 3]
```diff
@@ -15,22 +15,34 @@ export type WPLHDasboard = {
   playLevel: number;
   loveLevel: number;
   healthLevel: number;
-  reflection: string;
+  reflection1: string;
+  reflection2: string;
+  reflection3: string;
```
Comment (millasml) [inline src/types/Journal.ts:20]: ah, i think you gotta wait for the other branch to merge before you two can merge this one

**Review by millasml — CHANGES_REQUESTED** [review]
great job! some small comments :)

File: src/api/journal/createJournalEntry.ts (hunk 1 above)
Comment (ydymegan) [reply src/api/journal/createJournalEntry.ts]: alright done!

**Review by millasml — APPROVED** [review]
LGTM!

**Pull request merged**